src/orquestra/sdk/_base/_api/_wf_run.py

```diff
@@ -399,6 +400,38 @@ def get_results(self, wait: bool = False) -> t.Sequence[t.Any]:
 
         return results
 
+    def get_results_serialized(self, wait: bool = False) -> t.Sequence[WorkflowResult]:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏wdty about adding a new keyword only argument to `get_results` instead of a new method? maybe `serialized: bool = False`

Reply:
I could do that - but Im not super into that idea. Why? Coz basically the function returns enterily different data structure. Meaning we would have function that returns either <any> (as whatever WF function reruns) OR sequence of workflowResults.

TLDR: I could do that but I prefer the way it is. maybe @alexjuda has any strong opinions?

Reply:
I thought about a flag like this, but I think having a separate method is better. It avoids ambiguity in the returned type, usually leads to a better code completion, and avoids an `if` in our code + `assert isinstance(result, ...)` on the caller side.

Reply:
OK, how about using the `get_results_serialized` inside `get_results`? Just trying to minimise the number of locations we call the underlying runtime

Reply:
Absolutely makes sense, will do that